src/main.c

```diff
@@ -158,6 +166,27 @@ int main(int argc, char **argv) {
                 altitude_db_init((AltitudeDB *)packet_pos, last_time, 1000 * strtod(strtok(NULL, ":"), NULL));
                 break;
 
+            case DTYPE_ACCELERATON:
+                // if (!room_for_block(sizeof(AccelerationDB))) {
+                //     continue;
+                // }
+                just_added_block_size = sizeof(AccelerationDB);
+                add_block_header(DATA_ACCEL, just_added_block_size);
+                acceleration_db_init((AccelerationDB *)packet_pos, last_time, strtod(strtok(NULL, ","), NULL) / 100,
+                                     strtod(strtok(NULL, ","), NULL) / 100, strtod(strtok(NULL, ","), NULL) / 100);
+                break;
+
+            case DTYPE_VELOCITY:
+                // if (!room_for_block(sizeof(AngularVelocityDB))) {
```

Comment:
If you're not using this I'd just delete it tbh. I had assert statements there just to check for bugs, but checking for space will be added later when I come up with a better way of doing it lol